README.md

```diff
@@ -1,48 +1,98 @@
+> This is a web application that'll display the total number of COVID-19 cases in different countried round the world. The users can click on any country for more details.
 
+## Mockup
+This design was inspired by Nelson Sakwa, in association with the Creative Commons license.
+
+![Home page](./screenshot/Screenshot1-2022-09-02-85546.png)
+![Detail page](./screenshot/Screenshot2-2022-09-02-185721.png)
+
+## Video Presentation
+
+[watch video](https://www.loom.com/share/e27205a3b79e4866baf41b80c8a3697f)
+
+## Built With
+
+- HTML
+
+- CSS
+
+- JavaScript
+
+- React JS
+
+- Redux
+
+- Lighthouse (An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more).
+
+- Webhint (A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors).
+
+- Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
+
+- ESlint (A mighty, modern linter that helps you avoid errors and enforce conventions in JavaScript codes)
 
 ## Getting Started
 
-This repository includes files with plain SQL that can be used to recreate a database:
+To get a local copy up and running follow these simple example steps.
+
+- copy this link https://github.com/Roseokpe/covid-stats.git
+- get the directory that you want to clone the repository.
+- open the command prompt in this directory.
+- write git clone https://github.com/Roseokpe/covid-stats.git
+- go to the repository folder in your command prompt cd Metrics-webApp
+- use npm install to install the dependencies.
+- use npm start to start live server.
+
+## Prerequisites
+
+- Have a working and updated browser
+- Have a local version control like git install on your computer
+- Have an account and sign in on GitHub, as online/remote version control system
+- Basic knowledge in JS and React JS
+
+### Setup
+
+- Clone the git repo using this command: git clone https://github.com/Roseokpe/covid-stats.git
+
+### Run tests
 
-- Use [schema.sql](./schema.sql) to create all tables.
-- Use [data.sql](./data.sql) to populate tables with sample data.
-- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
+- npx hint . : To test html codes
+- npx stylelint "/*.{css,scss}"** : To test css code
+- npx eslint . : To test Eslint Report
 
+### To test the application
 
-## Authors
+- npm install --save-dev jest
+- npm test
 
-👤 **Author1**
+## Live Demo
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- [Live link](covid-metric-webapp.netlify.app)
```

Comment:
The link is not yet functional 😥 

![image](https://user-images.githubusercontent.com/15807343/188287420-ca5da005-0500-4477-b635-f574abccf43d.png)

Kindly make sure that the link in your readme is [live demo](https://covid19-globally.netlify.app) rather than [not functional](covid-metric-webapp.netlify.app)